doc/nrf/libraries/nrf_security/doc/driver_config.rst

```diff
@@ -463,26 +506,29 @@ To configure the Hash algorithms, set one or more of the following Kconfig optio
 Hash driver configurations
 ==========================
 
-You can use the following PSA driver-specific configurations for fine-grained control over which drivers provide the Hash algorithm.
-
-+-----------------------+---------------------------------------------------------------+---------------------------------------------------------------+
-| Hash algorithm        |  nrf_cc3xx driver support                                     | nrf_oberon driver support                                     |
-+=======================+===============================================================+===============================================================+
-| SHA-1 (weak)          | :kconfig:option:`CONFIG_PSA_CRYPTO_DRIVER_ALG_SHA_1_CC3XX`    | :kconfig:option:`CONFIG_PSA_CRYPTO_DRIVER_ALG_SHA_1_OBERON`   |
-+-----------------------+---------------------------------------------------------------+---------------------------------------------------------------+
-| SHA-224               | :kconfig:option:`CONFIG_PSA_CRYPTO_DRIVER_ALG_SHA_224_CC3XX`  | :kconfig:option:`CONFIG_PSA_CRYPTO_DRIVER_ALG_SHA_224_OBERON` |
-+-----------------------+---------------------------------------------------------------+---------------------------------------------------------------+
-| SHA-256               | :kconfig:option:`CONFIG_PSA_CRYPTO_DRIVER_ALG_SHA_256_CC3XX`  | :kconfig:option:`CONFIG_PSA_CRYPTO_DRIVER_ALG_SHA_256_OBERON` |
-+-----------------------+---------------------------------------------------------------+---------------------------------------------------------------+
-| SHA-384               | Not supported                                                 | :kconfig:option:`CONFIG_PSA_CRYPTO_DRIVER_ALG_SHA_384_OBERON` |
-+-----------------------+---------------------------------------------------------------+---------------------------------------------------------------+
-| SHA-512               | Not supported                                                 | :kconfig:option:`CONFIG_PSA_CRYPTO_DRIVER_ALG_SHA_512_OBERON` |
-+-----------------------+---------------------------------------------------------------+---------------------------------------------------------------+
-| MD5 (weak)            | Not supported                                                 | Not supported                                                 |
-+-----------------------+---------------------------------------------------------------+---------------------------------------------------------------+
-| RIPEMD-160            | Not supported                                                 | Not supported                                                 |
-+-----------------------+---------------------------------------------------------------+---------------------------------------------------------------+
-
+You can use the following table to check the Hash algorithm support of each driver:
```

Comment:
```suggestion
Use the following table to check the Hash algorithm support of each driver:
```

Please apply this change to other instances of this, where we changed 'you can use the following configurations' to 'you can check the table'.

Reply:
Search-replaced "You can use" with "Use" in the file